src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example: `font-family: Roboto, Arial, sans-serif;`